jenkins/ci.suse.de/cloud-mkcloud6.yaml

```diff
@@ -21,7 +21,6 @@
         - 'cloud-mkcloud{version}-job-hyperv-{arch}'
         - 'cloud-mkcloud{version}-job-raid-{arch}'
         - 'cloud-mkcloud{version}-job-ssl-{arch}'
-        - 'cloud-mkcloud{version}-job-upgrade-{arch}'
```

Comment:
This is deleting the job only from JJB. So no following JJB run will change this cloud6 upgrade job.